### PR TITLE
calc: set help example as lastest example

### PR DESCRIPTION
**Pull request by marchersimon**
#8148 has to be merged first

calc version 2.14.0.8

### CODE REVIEW

**Comment by navarroaxel** [comment]
@marchersimon please resolve conflicts.

File: pages/common/calc.md [hunk 1]
```diff
@@ -1,12 +1,36 @@
...
+> An interactive arbitrary-precision calculator in the terminal.
 > More information: <https://github.com/lcn2/calc>.
 
-- Start calc in interactive mode:
+- Start `calc` in interactive mode:
 
 `calc`
 
-- Perform a calculation in non-interactive mode:
+- Perform a calculation in non-interactive mode and exit:
```
Comment (EmilySeville7cfg) [inline pages/common/calc.md:10]:
```suggestion
- Perform a calculation in non-interactive mode:
```

If we assume that smth in non-interactive mode is exited by default then `and exit` is not necessary.

File: pages/common/calc.md [hunk 2]
```diff
@@ -1,12 +1,36 @@
...
+
+`calc -i '{{sqrt(2)}}'`
+
+- Start `calc` in a specific permission mode (0 to 7, defaults to 7):
+
+`calc -m {{mode}}`
+
+- Open the `calc` manual:
+
+`calc help`
```
Comment (EmilySeville7cfg) [inline pages/common/calc.md:28]: It's better to put this example to the bottom.

File: pages/common/calc.md [hunk 3]
```diff
@@ -1,12 +1,36 @@
...
+- Perform a calculation in non-interactive mode and exit:
 
-`calc -p '{{85 * (36 / 4)}}'`
+`calc '{{85 * (36 / 4)}}'`
+
+- Perform a calculation without any output formatting (for use with pipes):
+
+`calc -p '{{4/3 * pi() * 5^3}}'`
+
+- Perform a calculation and then switch to interactive mode:
```
Comment (EmilySeville7cfg) [inline pages/common/calc.md:18]:
```suggestion
- Perform a calculation and then switch to [i]nteractive mode:
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

**Comment by EmilySeville7cfg** [comment]
@marchersimon, you forgot to apply the same fixes to another page. 😄

**Comment by marchersimon** [comment]
At it, I'm currently trying to set up my Github GPG key for my new machine

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**